tests/test_cpu.py

```diff
@@ -201,6 +234,14 @@ def test_target_detection(expected_target):
     detected_target = archspec.cpu.host()
     assert detected_target == expected_target, f"{detected_target} == {expected_target}"
 
+def test_microarchitecture_name(expected_linux_target):
+    detected_target = archspec.cpu.host()
+    if expected_linux_target == 'thunderx2':
+        ## We don't have cpuid for thunderx2 yet
+        assert detected_target.name == "", "Unexpected non-empy target for thunderx2"
```

Comment:
:thinking: This would be a regression The cpupart for `thunderx2` is `0x0af` (it's in the sample used for tests).

Reply:
Would you just use the cpupart then? I think it is fine for us too.

Reply:
@paolotricerri Yes, see #177 as a tentative implementation.

Reply:
I'll ask for @dslarm's point of view and get back to you on that after our chat next Monday. Thanks